<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvUi</class>
 <widget class="QDialog" name="AdvUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Global Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="synthShortNames">
     <property name="toolTip">
      <string>Create smart shortened names. </string>
     </property>
     <property name="text">
      <string>Synthesize short names</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="enableCharSetXform">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Convert character set encoding between input and output</string>
     </property>
     <property name="text">
      <string>Enable character set transformation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="previewGmap">
     <property name="text">
      <string>Preview in Google Maps</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="debugCombo">
       <property name="toolTip">
        <string>Debugging diagnostics.  
Higher number provides more deitaled diagnostics.</string>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Debugging Diagnostics</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="formatButton">
       <property name="toolTip">
        <string>Set all format input/output options to default values</string>
       </property>
       <property name="text">
        <string>Default Format Options</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
